README.md

```diff
@@ -1,3 +1,33 @@
 # bevy_libgdx_atlas
 
-Support loading `libgdx` atlas files as bevy assets.
\ No newline at end of file
+[![Following released Bevy versions](https://img.shields.io/badge/Bevy%20tracking-released%20version-lightblue)](https://bevyengine.org/learn/quick-start/plugin-development/#main-branch-tracking)
+[![crates.io](https://img.shields.io/crates/v/bevy_libgdx_atlas)](https://crates.io/crates/bevy_libgdx_atlas)
+[![docs.rs](https://docs.rs/bevy_libgdx_atlas/badge.svg)](https://docs.rs/bevy_libgdx_atlas)
+
+
+
+Support loading `libgdx.atlas` files (used for sprite sheets and such) as Bevy assets.
+
+## Usage
+
+Add the `LibGdxAssetPlugin` to your app:
+
+```rust
+app.add_plugins(LibGdxAssetPlugin);
+```
+
+Now when you load files with the `.libgdx.atlas` extension through the asset server, or even `bevy_asset_loader`, they will load as a `LibGdxAtlasAsset` which you can then use.
```

Comment:
maybe a snippet on how to change the frame of a sprite using `LibGdxAtlasAsset` (fine in a follow up PR)

Reply:
You mean like?
```rust
if let Some(index) = libgdx_atlas.files.get("01".into()) {
    texture_atlas.index = index;
}
```

Reply:
yeah plus a comment explaining it (follow up PR)